xlgui/collection.py

```diff
@@ -64,6 +65,17 @@ def __init__(self, parent, collection):
         for location, library in collection.libraries.items():
             self.model.append([location, library.monitored, library.startup_scan])
 
+        # Override the data function for location_column, so that it
+        # displays unespaced location URIs instead of escaped ones
+        def display_unescaped_location(column, cell, model, iter, *data):
```

Comment:
Rather than simply unescaping the URL, I think it would be nicer to use [`Gio.File.get_parse_name`](https://lazka.github.io/pgi-docs/#Gio-2.0/classes/File.html#Gio.File.get_parse_name), which I believe displays local paths when possible.

Reply:
It is indeed a better choice. Thanks!